.circleci/config.yml

```diff
@@ -99,104 +99,6 @@ commands:
             - ~/.cache/go-build
             - /go/pkg/mod
       - run: make fmt-check
-  build-macos:
-    steps:
-      - checkout
-      - submodules
-      - run:
-          name: "Install dependencies"
-          command: |
-            curl https://dl.google.com/go/go1.17.darwin-amd64.tar.gz -o go1.17.darwin-amd64.tar.gz
-            sudo tar -C /usr/local -xzf go1.17.darwin-amd64.tar.gz
-            ln -s /usr/local/go/bin/go /usr/local/bin/go
-            HOMEBREW_NO_AUTO_UPDATE=1 brew install cmake ninja
```

Comment:
These things should still all be installed unconditionally, as the logic is a bit complicated to which we need